server/src/main/kotlin/com/xebia/functional/xef/server/http/routes/Requests.kt

```diff
@@ -0,0 +1,38 @@
+package com.xebia.functional.xef.server.http.routes
+
+import io.ktor.resources.*
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ChatMessage(
+    val role: ChatRole, val content: String? = null, val name: String? = null
+)
+
+@JvmInline
+@Serializable
+value class ChatRole(val role: String) {
+    public companion object {
+        public val System: ChatRole = ChatRole("system")
+        public val User: ChatRole = ChatRole("user")
+        public val Assistant: ChatRole = ChatRole("assistant")
+        public val Function: ChatRole = ChatRole("function")
+    }
+}
+
+
+@Serializable
+@Resource("/chat/completions")
+data class PromptMessageRequest(
```

Comment:
We are not using `PromptMessageRequest` in our code. Are we planning to use it in the future?

Reply:
I think the three models are not needed, you are right. Removed, thanks!